src/main/java/com/amazon/opendistroforelasticsearch/search/asynchronous/plugin/AsynchronousSearchPlugin.java

```diff
@@ -85,6 +85,7 @@
 
     public static final String OPEN_DISTRO_ASYNC_SEARCH_GENERIC_THREAD_POOL_NAME = "opendistro_asynchronous_search_generic";
     public static final String BASE_URI = "/_opendistro/_asynchronous_search";
```

Comment:
I would rename this to `OPENDISTRO_BASE_URI` for consistency and avoiding confusion on which one to use for the future.